### DISCUSSION TITLE
Osmand Discontinuing Development???

### DISCUSSION THREAD

**Discussion by ThisIsEd** [osmandapp/OsmAnd#19939] [General]
December last year I tried to contact Osmand developers. Sent emails, opened a ticket with the intention to **PAY** for support. That's right, **PAY!**. I asked for the cost, paypal ready, for a 10 minute phone call as I needed expedited support in using the API. They recommended to use the forum. So I posted this about a  month ago:
[https://github.com/osmandapp/OsmAnd/discussions/19633](url)
No response.
I've sent several support requests. Got this once:
"OsmAnd Support (Support Team)

Apr 25, 2024, 17:49 GMT+3
Hello,
 
We sincerely apologise for the long wait for a reply, we have a new release at the moment and so the developers are very busy. 
You will be answered as soon as possible.
 
Regards,
Zinaida"

Replied to it several times. No response since then.

Is this project now abandonware???

**If money is not a driving force for anyone at Osmand, Will they be there tomorrow? or Will we be forced to figure out how to export all of our data to some other app?** 

**Does anyone know what's going on with this project? Should we start looking for something else already?**

**Comment by onetimecontributor** [comment]
That is so creepy to post it) "PAY", LoL. The prj looks fine at least for me from the outside. Releases are flowing, the tool is improving. Thanks to the team.

**Comment by onetimecontributor** [comment]
Back to kinda constructive mood - it looks like Discussions here at GitHub are not ?yet popular enough as of community support. You may want to explore some other channels.

**Comment by vshcherb** [comment]
There is no special paid support, I guess it's only a part of support discussion and probably solution was not found or the case was lost 

Though you can continue this post and describe the issue publicly.

**Reply by vshcherb** [reply]
Actually will reply https://github.com/osmandapp/OsmAnd/discussions/19633.
